src/routes/boot.ts

```diff
@@ -124,12 +132,17 @@ const excludeProperties = <T, K extends keyof T>(
   obj: T,
   properties: K[],
 ): Pick<T, Exclude<keyof T, K>> => {
-  if (obj) {
-    properties.forEach((prop) => {
-      delete obj[prop];
-    });
+  if (!obj) {
+    return obj;
   }
-  return obj;
+
+  const clone = structuredClone(obj);
+
+  properties.forEach((prop) => {
+    delete clone[prop];
+  });
+
+  return clone;
```

Comment:
Had to update since JS in nature would use the same object referenced from the parameters, hence affecting the original object where it came from. And on the other side, we still need that property for some calculations. With that, we you run a function to modify an object, you normally expect it to not affect the original object and return a new one instead.